packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/mainTreeListMenu..st

```diff
@@ -0,0 +1,10 @@
+as yet unclassified
+mainTreeListMenu: aMenu
+	<treeListMenu>
```

Comment:
Could you explain this syntax to me? Didn’t see it before and tried to find out about it online, but without luck… 😢

Reply:
We unfortunately cant. This is the way the `Object-Explorer` implements the menu.

Reply:
Ich verstehe die Syntax leider auch nicht :(